### DISCUSSION TITLE
Image_load error

### DISCUSSION THREAD

**Discussion by lh13329089918** [nothings/stb#1698] [Q&A]
<img width="632" alt="捕获" src="https://github.com/user-attachments/assets/7c62956c-05b4-420e-8fa0-d8a997b10a37">
I've checked the path of path, but stbi_load always returns NULL. What's wrong?

**Comment by nothings** [comment]
Call `stbi_failure_reason()` to get a string explaining why it failed. If the message is unclear, #define STBI_FAILURE_USERMSG before creating the implementation to get more verbose error messages.

**Reply by lh13329089918** [reply]
The reason is actually an unknown type, even though my image is in PNG format.
<img width="360" alt="捕获1" src="https://github.com/user-attachments/assets/c8d4c0bf-c45b-43d1-b4df-53217b81bcb0">
